posthog/heatmaps/sql_two.py

```diff
@@ -0,0 +1,51 @@
+from django.conf import settings
+
+from posthog.clickhouse.kafka_engine import kafka_engine
+from posthog.kafka_client.topics import KAFKA_CLICKHOUSE_HEATMAP_EVENTS
+
+HEATMAPS_DATA_TABLE = lambda: "sharded_heatmaps"
+
+# using JSONAsString we can send multiple JSON items in a single kafka messages,
+# but we have to write to a String column
+# JSONEachRow checks that the payload it is receiving starts with '{'
+KAFKA_HEATMAPS_TABLE_BASE_SQL = """
+CREATE TABLE IF NOT EXISTS {table_name} ON CLUSTER '{cluster}'
+(
+    kafka_payload String
+) ENGINE = {engine}
+"""
```

Comment:
so, with `JSONAsString` if we send `[{a: 1}, {a: 2}, {a: 3}]`

the kafka table will have three rows

```
"{\"a\": 1}"
"{\"a\": 2}"
"{\"a\": 3}"
```